.github/ISSUE_TEMPLATE/001-bug-report.yml

```diff
@@ -0,0 +1,99 @@
+name: Bug Report
+description: Report a bug where something is not working as expected, which does not crash the game.
+labels: [ "status: unverified", "bug: gameplay" ]
+body:
+  - type: input
+    id: version
+    attributes:
+      label: GregTech CEu Version
+      description: The version of Gregtech CEu you were using when this bug was encountered. If you do not know where to find this, look for the mod jar file in the mods folder, or in the mods viewer in-game.
+      placeholder: "Example: v2.3.4"
+    validations:
+      required: true
+  - type: input
+    id: modpack
+    attributes:
+      label: Modpack Used
+      description: The modpack and its version used when this bug was encountered. If you were not using a modpack, please leave this field blank.
+      placeholder: "Example: GregTech Community Modpack v1.2.1"
+    validations:
+      required: false
+  - type: textarea
+    id: addons
+    attributes:
+      label: Addons Installed
+      description: The addons and their respective versions used when this bug was encountered. If you were not using addons, or do not know which are present (i.e. in a modpack), please leave this field blank.
+      placeholder: "Example: Gregicality Multiblocks v1.1.3"
+    validations:
+      required: false
+  - type: dropdown
+    id: environment
+    attributes:
+      label: Environment
+      description: How you were playing on the world. Open to LAN means you did not start the world with dedicated server software, and used the in-game pause menu to start it.
+      options:
+        - Singleplayer
+        - Open to LAN
+        - Base Forge Server
+        - Sponge
+        - Mohist
+        - Other
```

Comment:
again, any way to have an option to have a specific value for `other` with a text box